_data/harnesses/multi-joins/ksql.yml

```diff
@@ -0,0 +1,132 @@
+dev:
```

Comment:
@rspurgeon would you consider adding a `Short Answer` to this KT? (e.g. https://kafka-tutorials.confluent.io/create-stateful-aggregation-count/ksql.html)

Reply:
added in latest